CITATION.cff

```diff
@@ -3,6 +3,11 @@ title: pywaterinfo
 message: 'If you use this software, please cite it as below.'
 type: software
 authors:
+  - given-names: Stijn
+    family-names: ' Van Hoey'
+    email: stijn@fluves.com
+    affiliation: Fluves
+    orcid: 'https://orcid.org/0000-0001-6413-3185'
   - given-names: Johan
     name-particle: 'van de '
     family-names: Wauw
```

Comment:
Correct way would be "Van de Wauw" as family name I think. In any case the V has a capital and the d does not

Reply:
aha, indeed; I taught both are lowercase. Sorry, my mistake.